templates/zerver/help/animated-gifs-from-giphy.md

```diff
@@ -37,6 +37,20 @@ this configure or disable GIPHY integration entirely:
 
 {end_tabs}
 
+## Privacy
+
+GIPHY is a third-party service owned by Facebook, and the text you
+enter into the GIPHY search box is sent by your browser to GIPHY's
+servers via their API. Zulip does a few things to help protect users'
+privacy:
+
+* Zulip proxies all external images in messages through the server,
+  including those from GIPHY, to prevent images from being used to
+  track you.
+* Zulip queries the GIPHY API without providing any identifier for you
+  as a user. As a result, the GIPHY search results are not
+  personalized to you.
```

Comment:
This seems a bit misleading, or there's a bug with the feature. Or I'm misunderstanding things. All are possible, I'll try to explain why I think it's misleading:

When I click on the GIF button, it tries to connect to `api.giphy.com` directly. That request is not proxied by Zulip, therefore Facebook gains access to a personal identifier: my IP address. (The request is blocked by umatrix locally, hence it fails in the network tab)

![grafik](https://user-images.githubusercontent.com/20866761/118329133-ef0d7580-b506-11eb-8760-931bd4722b3c.png)

Further, uMatrix shows me that `giphy.com` attempts to do something with cookies: 

![grafik](https://user-images.githubusercontent.com/20866761/118329350-2a0fa900-b507-11eb-822a-2fc04451aa55.png)

I don't know how accurate that is, however. Sometimes uMatrix' request log is a bit unreliable.

---

Could you clarify if this is the right behaviour, and if so, perhaps clarify the note above?

I'm happy to provide further info for debugging in case this is not the intended behaviour, but you'll have to tell me what you need.

Reply:
Yeah, I tried to express that with the "your browser sends", but I think it's too subtle, and a result maybe not worth documenting our avoiding passing their explicit personalization key parameter.

Reply:
OK, I pushed a revised version that I think explains this correctly, with the phrase "similar to if you visited the
GIPHY website and typed the same search keywords there", which I think is the right comparison.

Does the new phrasing seem right to you?

Reply:
> Does the new phrasing seem right to you?

Yes, it seems better. Thank you for addressing it.

It makes the difference clearer that using the integration to post gifs will involve accessing FB services whereas merely reading messages with gifs in them won't.

Reply:
Great, thanks for the feedback!